### PR TITLE
Add `Asset.contractId()` helper to predict contract IDs

### PR DESCRIPTION
**Pull request by Shaptic**
This just needs a few more test cases before it's ready to merge, cc @orbitlens.

### CODE REVIEW

**Review by sreuland — APPROVED** [review]
nice utility.

**Review by piyalbasu — APPROVED** [review]

File: test/unit/asset_test.js [hunk 1]
```diff
@@ -279,104 +277,121 @@ describe('Asset', function () {
...
+  });
+
+  describe('contractId()', function () {
+    it('creates the correct contract IDs', function () {
+      [
+        [
+          Asset.native(),
+          'CB64D3G7SM2RTH6JSGG34DDTFTQ5CFDKVDZJZSODMCX4NJ2HV2KN7OHT'
+        ]
+        // TODO: Add more
```
Comment (orbitlens) [inline test/unit/asset_test.js:386]: Test-case for alpha-num4 asset:

```suggestion
        [
          Asset.native(),
          'CB64D3G7SM2RTH6JSGG34DDTFTQ5CFDKVDZJZSODMCX4NJ2HV2KN7OHT'
        ],
        [
          new Asset('USD', 'GCP2QKBFLLEEWYVKAIXIJIJNCZ6XEBIE4PCDB6BF3GUB6FGE2RQ3HDVP'),
          'CCWNZPARJG7KQ6N4BGZ5OBWKSSK4AVQ5URLDRXB4ZJXKGEJQTIIRPAHN'
        ]
```

**Comment by orbitlens** [comment]
Wow, that was super fast, thanks for adding this functionality to the SDK. Above, I also added my test-case for alphanum4 asset.

File: test/unit/asset_test.js (hunk 1 above)
Comment (Shaptic) [reply test/unit/asset_test.js]: perfect, thanks!

**Pull request merged**